frontend/src/components/common/table/table.tsx

```diff
@@ -90,11 +90,11 @@ const Table: FC<Props> = ({
               );
             })}
           </tbody>
+          {hasPlaceholder && (
+            <div className={styles.placeholder}>{placeholder}</div>
+          )}
```

Comment:
inside `body` we can't have the `div`
https://caninclude.glitch.me/

Reply:
Thanks
Maybe `table`?

Reply:
right!